test/celeritas/data/testem3-flat.gdml

```diff
@@ -4,1288 +4,1288 @@
-      <physvol name="absorber_lv_4">
-        <volumeref ref="absorber_lv_4"/>
-        <position name="absorber_lv_4_pos" unit="mm" x="-162.85" y="0" z="0"/>
-      </physvol>
-      <physvol name="gap_lv_5">
-        <volumeref ref="gap_lv_5"/>
-        <position name="gap_lv_5_pos" unit="mm" x="-158.85" y="0" z="0"/>
-      </physvol>
-      <physvol name="absorber_lv_5">
-        <volumeref ref="absorber_lv_5"/>
-        <position name="absorber_lv_5_pos" unit="mm" x="-154.85" y="0" z="0"/>
-      </physvol>
-      <physvol name="gap_lv_6">
-        <volumeref ref="gap_lv_6"/>
-        <position name="gap_lv_6_pos" unit="mm" x="-150.85" y="0" z="0"/>
-      </physvol>
-      <physvol name="absorber_lv_6">
-        <volumeref ref="absorber_lv_6"/>
-        <position name="absorber_lv_6_pos" unit="mm" x="-146.85" y="0" z="0"/>
-      </physvol>
-      <physvol name="gap_lv_7">
-        <volumeref ref="gap_lv_7"/>
-        <position name="gap_lv_7_pos" unit="mm" x="-142.85" y="0" z="0"/>
-      </physvol>
-      <physvol name="absorber_lv_7">
-        <volumeref ref="absorber_lv_7"/>
-        <position name="absorber_lv_7_pos" unit="mm" x="-138.85" y="0" z="0"/>
-      </physvol>
-      <physvol name="gap_lv_8">
-        <volumeref ref="gap_lv_8"/>
-        <position name="gap_lv_8_pos" unit="mm" x="-134.85" y="0" z="0"/>
-      </physvol>
-      <physvol name="absorber_lv_8">
-        <volumeref ref="absorber_lv_8"/>
-        <position name="absorber_lv_8_pos" unit="mm" x="-130.85" y="0" z="0"/>
-      </physvol>
-      <physvol name="gap_lv_9">
-        <volumeref ref="gap_lv_9"/>
-        <position name="gap_lv_9_pos" unit="mm" x="-126.85" y="0" z="0"/>
-      </physvol>
-      <physvol name="absorber_lv_9">
-        <volumeref ref="absorber_lv_9"/>
-        <position name="absorber_lv_9_pos" unit="mm" x="-122.85" y="0" z="0"/>
-      </physvol>
-      <physvol name="gap_lv_10">
-        <volumeref ref="gap_lv_10"/>
-        <position name="gap_lv_10_pos" unit="mm" x="-118.85" y="0" z="0"/>
-      </physvol>
-      <physvol name="absorber_lv_10">
-        <volumeref ref="absorber_lv_10"/>
-        <position name="absorber_lv_10_pos" unit="mm" x="-114.85" y="0" z="0"/>
-      </physvol>
-      <physvol name="gap_lv_11">
-        <volumeref ref="gap_lv_11"/>
-        <position name="gap_lv_11_pos" unit="mm" x="-110.85" y="0" z="0"/>
-      </physvol>
-      <physvol name="absorber_lv_11">
-        <volumeref ref="absorber_lv_11"/>
-        <position name="absorber_lv_11_pos" unit="mm" x="-106.85" y="0" z="0"/>
+    <volume name="gap_00x600001b3cdc0">
+      <materialref ref="Pb0x14ce28a60"/>
+      <solidref ref="gap_shape0x60000013ce70"/>
+      <auxiliary auxtype="gammaECut" auxunit="MeV" auxvalue="0.0946952"/>
+      <auxiliary auxtype="electronECut" auxunit="MeV" auxvalue="1.00699"/>
+      <auxiliary auxtype="positronECut" auxunit="MeV" auxvalue="0.946936"/>
+      <auxiliary auxtype="protonECut" auxunit="MeV" auxvalue="0.07"/>
```

Comment:
More a question about the geometry exporter, but since the Geant4 GDML parser doesn't actually read/apply these cutoff values, is there any reason to export them? It seems like it might potentially cause confusion and bloat the GDML file.

Reply:
That is a good point. I guess we'd have to read these in the SD layer and later reuse those stored values in the physics? Sounds like a lot of work though, especially if no one uses them. @stognini this is your department: maybe we shouldn't save these?

Reply:
We never fully implemented the import of these cutoff values, but I added them early in the exporter to have that ready. We could just set the export cutoff flag to false when exporting the gdmls until we're ready to use them.

Reply:
Just set `G4GDMLParser::SetEnergyCutsExport` flag to `false` in [utils/gdml-generator](https://github.com/celeritas-project/utils/tree/main/gdml-generator) for the time being.